src/status_im/ui/screens/chat/styles/main.cljs

```diff
@@ -238,6 +238,29 @@
   {:margin-bottom 5
    :color         colors/black})
 
+(def empty-chat-name
+  {:style {:font-size     22
+           :font-weight   :bold
+           :line-height   28
+           :margin-bottom 8
+           :color         colors/black}})
+
+(defn empty-chat-icon [diameter color]
+  {:margin-bottom    24
+   :width            diameter
+   :height           diameter
+   :align-items      :center
+   :justify-content  :center
+   :border-radius    (/ diameter 2)
+   :background-color color})
+
+(def empty-chat-icon-text
+  {:style {:color       colors/white
```

Comment:
could you please move style key outside?

Reply:
done!